proposals/tx-size.md

```diff
@@ -48,18 +48,54 @@ get bodySize() {
 }
 ```
 
+### `FileAppendTransaction.bodySizeAllChunks`
```

Comment:
Should this be just for `FileAppendTransaction` or for all `ChunkedTransaction`s (looking at `TopicMessageSubmitTransaction` as well)?

Reply:
It should work on TopicMessageSubmitTransaction as well.

Reply:
Will change to document to show that it should work on ChunkedTransaction